packages/vue/src/components/molecules/SfScrollable/SfScrollable.vue

```diff
@@ -22,18 +22,38 @@ export default {
     Simplebar,
     SfButton,
   },
+  inject: {
+    contentShow: {
+      default: "contentShow",
+    },
+  },
   props: {
     maxContentHeight: {
       type: String,
-      default: "",
+      default() {
```

Comment:
@AdamPawlinski IMO you cant get data for default from anywhere 
![Zrzut ekranu 2020-05-8 o 13 17 00](https://user-images.githubusercontent.com/12138170/81400835-43c92400-912e-11ea-894b-aee852997609.png)

Reply:
and you shouldn't change SfScrollable, just use props